Modules/socketmodule.c

```diff
@@ -1616,7 +1621,7 @@ makesockaddr(SOCKET_T sockfd, struct sockaddr *addr, size_t addrlen, int proto)
 #ifdef CAN_ISOTP
           case CAN_ISOTP:
           {
-              return Py_BuildValue("O&kk", PyUnicode_DecodeFSDefault,
+              return Py_BuildValue("O&kk", unicode_fsdecode,
                                           ifname,
```

Comment:
Let's also take the opportunity to align `ifname` and the rest with `unicode_fsdecode`